src/domain/occurrence/locationFormPart/LocationFormPart.tsx

```diff
@@ -43,19 +44,28 @@ const LocationFormPart: React.FC<{ selectedLanguages: Language[] }> = ({
         <div>
           <h2>{t('eventForm.location.title')}</h2>
           <div className={styles.locationSection}>
-            <Field
-              label={t('eventForm.location.labelEventIsHeldVirtual')}
-              name="isVirtual"
-              component={VirtualEventCheckboxField}
-            />
+            <div className={styles.locationCheckboxes}>
+              <Field
+                label={t('eventForm.location.labelEventIsHeldVirtual')}
+                name="isVirtual"
+                disabled={isOrderable}
+                component={VirtualEventCheckboxField}
+              />
+              <Field
+                label="Tapahtuma on tilattavissa päiväkotiin tai kouluun"
```

Comment:
Needs translations

Reply:
Ticket said that we aren't even sure if this is the correct label we are going to use so I didn't get translation for it yet